packages/gluestick/src/types.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+
+export type Context = {
+ config: {
+   projectConfig?: ProjectConfig;
+   GSConfig?: GSConfig;
+   webpackConfig?: WebpackConfig;
+   plugins: Array<Object>;
+ },
+ logger: {
+   success: Function; // Function: (...args: string) => void
```

Comment:
`Function` is a really weak type, it means it can be any type of function. Better approach would be (didn't test it):

```
type LoggerTypes = 'success'
  | 'info'
  | 'warn'
  | 'debug'
  | 'error';
```

```
(type: LoggerTypes) => (args: Array) => void;
// Same for others
```